scRNA-seq-advanced/01-read_filter_normalize_scRNA.Rmd

```diff
@@ -79,75 +79,93 @@ ref_dir <- file.path("data", "reference")
 
 # Path to mitochondrial genes table
 mito_file <- file.path(ref_dir, "hs_mitochondrial_genes.tsv")
+```
 
-
+```{r outputs}
 # Outputs ------------------------------------
 
 # Directory and file to save output
 normalized_dir <- file.path(data_dir, "normalized")
+
+# create the directory if it does not exist 
 if (!dir.exists(normalized_dir)) {
   dir.create(normalized_dir, recursive = TRUE)
 }
 
+# output RDS file for normalized data
 output_sce_file <- file.path(normalized_dir, "gliobastoma_normalized_sce.rds")
 ```
 
 
 ## Reading Cell Ranger data
 
 Whether the 10X Cell Ranger data is in Matrix Exchange format or in an HDF5 file, we can use the `read10xCounts()` function from the `DropletUtils` package to read the data into R and create a `SingleCellExperiment` object.
+(If you used something other than Cell Ranger to process the raw data, you would need to use a different function to read it in and create the `SingleCellExperiment` object.)
```

Comment:
Might be worth linking this section - http://bioconductor.org/books/3.16/OSCA.intro/getting-scrna-seq-datasets.html#reading-counts-into-r